README.md

```diff
@@ -1,105 +1,82 @@
-<p align="center">
-  <a href="https://github.com/actions/typescript-action/actions"><img alt="typescript-action status" src="https://github.com/actions/typescript-action/workflows/build-test/badge.svg"></a>
-</p>
+# keyvault-yaml-secret
 
-# Create a JavaScript Action using TypeScript
+In teacher services we store our secrets in Azure Key Vault. To make consumption easier, common application secrets are stored as YAML strings within a Key Vault secret object.
```

Comment:
We could make it more wide than teacher services. You can point to https://technical-guidance.education.gov.uk/infrastructure/security/managing-secrets/ as it's a pattern in DfE

Reply:
Nice!! Sounds good.